frame/contracts/src/weights.rs

```diff
@@ -230,25 +230,25 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: System Account (r:1 w:1)
 	fn call() -> Weight {
-		(140_169_000 as Weight)
+		(187_520_000 as Weight)
```

Comment:
General `call` overhead increased quite a bit. Most likely setting up a new module and instance just takes longer in the new version. Do we have a benchmark for this in wasmi (creating a new instance) @Robbepop ?

Reply:
yes there are benchmarks for checking the efficiency of module instantiation.